src/Controller/CampusController.php

```diff
@@ -9,10 +9,12 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Attribute\Route;
+use Symfony\Component\Security\Http\Attribute\IsGranted;
 
 class CampusController  extends AbstractController
 {
     #[Route('/campus', name: 'app_campus')]
+    #[IsGranted('ROLE_ADMIN')]
     public function index(CampusRepository $campusRepository): Response
     {
         $campus = $campusRepository->findAll();
```

Comment:
Yannis Sileber
📷

c'est normal que la methode $this->denyAccessUnlessGranted("ROLE_ADMIN"); soit pas dans le campusController aussi ?